app/Models/Job.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Job extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
+        'keywords',
+        'title',
+        'location',
+        'type'
+    ];
+
+    public function jobs() {
+        return $this->hasOne(Employer::class);
```

Comment:
If I’m not wrong this is the inverse relationship, which means that must be “belongsTo” instead